ambassador/ambassador/envoy/v2/v2route.py

```diff
@@ -24,14 +25,31 @@
 
 
 class V2Route(dict):
-    def __init__(self, config: 'V2Config', group: IRHTTPMappingGroup) -> None:
+    def __init__(self, config: 'V2Config', group: IRHTTPMappingGroup, mapping: IRBaseMapping) -> None:
         super().__init__()
 
         envoy_route = EnvoyRoute(group).envoy_route
 
+        mapping_prefix = mapping.get('prefix', None)
+        route_prefix = mapping_prefix if mapping_prefix is not None else group.get('prefix')
+
+        mapping_case_sensitive = mapping.get('case_sensitive', None)
+        case_sensitive = mapping_case_sensitive if mapping_case_sensitive is not None else group.get('case_sensitive', True)
+
```

Comment:
Wow. We really need to set up a method that gets from a mapping with a fallback to the group, huh?

Reply:
@kflynn so, I've added fallback to group to the fields that we are playing with in the group. So, just in case the mapping does not have field _x_ but the group has determined that the mapping _should_ have _x_, then we accept _x_.

priority(mapping) > priority(group)

Makes sense?